jekyll/_cci1/test-metadata.md

```diff
@@ -156,6 +157,21 @@ test:
 
 
 
+#### <a name="eslint"></a>ESLint
+
+To output JUnit results from [ESLint](http://eslint.org/), you can use the [JUnit formatter](http://eslint.org/docs/user-guide/formatters/#junit).
+
+A working [circle.yml]( {{ site.baseurl }}/1.0/configuration/) section for testing might look like this:
```

Comment:
Edit:

A working [circle.yml]( {{ site.baseurl }}/1.0/configuration/) `test` section might looke like this: